packages/portal-marketmaker/src/index.js

```diff
@@ -0,0 +1,407 @@
+const axios = require('axios');
+const { ethers } = require("ethers");
+const bitcore = require("bitcore-lib");
+const btcproof = require("bitcoin-proof");
+var bitcoinjs = require('bitcoinjs-lib');
+
+const satsToBTC = 10**8;
+const wbtcAddress = "0xBde8bB00A7eF67007A96945B3a3621177B615C44"
+const wbtcAbi = [
+	"function balanceOf(address) external view returns (uint256)",
+	"function allocateTo(address,uint256)",
+];
+const portalAddress = "0xa909c8B6eD96899dFb82a698FEF380e8836e00b9";
+
+const ethRpcProvider = "https://ropsten.infura.io/v3/6f2cc3019abd4ffa8045a331fc47f3d4"
+const privateKey = "0ccd0dfe2df9ae2c9e325fcb09f0bf2101eae834241a5812d81e015c7d539443"
+const myAddress = "0xb1032f1330d3d4db2db412e34050482e2fc756f1"
+const maker = "0xb1032f1330d3d4db2db412e34050482e2fc756f1";
+const status = "PENDING";
+const myOrdersQuery = `{orders(maker: $maker, status: $status) {
+  amountSats
+  priceTokPerSat
+}}`;
+
+/*
+// orders that we want to fulfill because we can make money off them
+const ordersToFulfill = `{orders(where: {maker_not_contains_nocase : $maker, amountSats_lt : 0, priceTokPerSat_gt : $minPrice}) {
+	amountSats
+	priceTokPerSat
+}}`
+const minPrice = 9950000000;
+*/
+
+const escrowsQuery = `{escrows(where : {successRecipient_contains: "0xb1032f1330d3d4db2db412e34050482e2fc756f1", status: "PENDING"}) {
+	amountSatsDue
+	destScriptHash
+	deadline
+	id
+}_meta { block { number }}}`;
+
+const myBtcAddress = "mqHEJi3boWQHwo2x8fEpXejyDWj37yTtDG";
+const myBtcKey = "cNdotpEDrvEhCp7rABaent5cqnvrX9gGph3eL1PPQLt3AxmAwETh";
+//const myBtcAddress = "0x529ccdd3112490bc59754892787e93f124520e78";
+
+var bidSats = BigInt(0);
```

Comment:
give these globals one-liner comments

is `bidSats` the total amount of all open bids?

if a bid gets partially filled, does bidSats go down by that amount?